app/models/cinstance.rb

```diff
@@ -200,7 +201,7 @@ def self.by_service(service)
   # maybe move both limit methods to their models?
 
   def self.serialization_preloading
-    includes(:application_keys, :plan, :user_account,
+    includes(:plan, :user_account,
```

Comment:
I had a Bullet error without this. 
```
Bullet::Notification::UnoptimizedQueryError: user: avalon
GET /admin/api/applications
AVOID eager loading detected
  Cinstance => [:application_keys]
  Remove from your query: .includes([:application_keys])
```

FYI this can be worked  around by:
```
Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :application_keys
```
but this is not the point. More interesting is why it didn't appear previously... maybe because things were obtained by `JOIN`, not by separate queries.

Either way, seems unused.

Reply:
Many tests fail because of this error, I assume it's already fixed?

Reply:
There are more, not only this. But this was hiding the others.

Reply:
All related should be fixed now. I've spent a lot of time to make sure this calls preloads everything. While I've just fixed whatever failed for the rest.